autotask/pricers-bot/codebuild/derived-asset/index.js

```diff
@@ -0,0 +1,92 @@

```

Comment:
is it possible to import these from Gamma since they are already in this repo or do they need to be pasted here? :)

Reply:
Yes they need to be pasted there. It's the template file the autotask-client package copies and pastes on the defender bot.